src/Dict/Any.elm

```diff
@@ -573,3 +573,47 @@ encode keyE valueE =
     Encode.object
         << List.map (Tuple.mapBoth keyE valueE)
         << toList
+
+
+{-| Turn an AnyDict into a JSON list of tuples. This is useful when you have more complex types as keys
```

Comment:
I think mentioning `tuples` in JSON context might be a bit confusing since tuple doesn't context in JSON so in JSON terms this is a two dimensional array.